app/models/proxy_order.rb

```diff
@@ -33,7 +33,7 @@ def canceled?
   end
 
   def cancel
-    return false unless order_cycle.orders_close_at.andand > Time.zone.now
+    return false unless order_cycle.orders_close_at&.send(:>, Time.zone.now)
```

Comment:
Why not just this?

```suggestion
    return false unless order_cycle.orders_close_at&. > Time.zone.now
```

Reply:
Ah nice, I hadn't realised that would work :+1:

Reply:
I adjusted the formatting to avoid syntax errors and comply with the style guide. This doesn't work:
```rb
subscription.present? && order_cycle.orders_close_at&.> Time.zone.now
# E: Lint/Syntax: unexpected token tCONSTANT
#      subscription.present? && order_cycle.orders_close_at&.> Time.zone.now
#                                                              ^^^^
```